terraform/telemetry.py

```diff
@@ -0,0 +1,39 @@
+from google.cloud import pubsub_v1
+from datetime import datetime
+import sys
+
+project_id = "stackdriver-sandbox-230822"
+topic_id = "telemetry"
+
+publisher = pubsub_v1.PublisherClient()
+# The `topic_path` method creates a fully qualified identifier
+# in the form `projects/{project_id}/topics/{topic_id}`
+topic_path = publisher.topic_path(project_id, topic_id)
```

Comment:
how authentication will work? Will everyone be able to send telemetry from any project into our stackdriver-sandbox-230822 project?

Reply:
Currently, I am running this locally with my service account (in which I download and store GOOGLE_APPLICATION_CREDENTIALS). I'm not sure how that will translate when everyone is working on it. I'll look into it.

Reply:
Instead of embedding a service account as we discussed today, could we just make the topic open to `allAuthenticatedUsers`? That should be essentially the same security, and then we don't have to worry about security reviews, etc